src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { Loader } from '../Loader';
+import { PersonLink } from '../PersonLink';
+import classNames from 'classnames';
+import { Person } from '../../types';
+
+interface PeopleTableProps {
+  people: Person[] | null;
+  isLoading: boolean;
+  error: boolean;
+  selectedPersonName: string | null;
+}
+
+export const PeopleTable: React.FC<PeopleTableProps> = ({
+  people,
+  isLoading,
+  error,
+  selectedPersonName,
+}) => {
+  const getPerson = (data: Person[], name: string | null) => {
+    if (!name) {
+      return null;
+    }
+
+    const person = data.find(item => item.name === name);
+
+    return person || null;
+  };
+
+  const renderPersonCell = (data: Person[], name: string) => {
+    const person = getPerson(data, name);
+
+    return person ? <PersonLink person={person} /> : name;
+  };
```

Comment:
Would be better to prepare your people array before rendering and to add `mother` and `father` to every person object